pkg/machinery/config/configloader/internal/decoder/decoder.go

```diff
@@ -20,8 +20,8 @@ import (
 var ErrMissingKind = errors.New("missing kind")
 
 const (
-	// ManifestVersionKey is the string indicating a manifest's version.
-	ManifestVersionKey = "version"
+	// ManifestAPIVersionKey is the string indicating a manifest's version.
+	ManifestAPIVersionKey = "apiVersion"
```

Comment:
isn't this a breaking change, i mean for existing config?

Reply:
nope, we never ever actually supported multi-doc before